zetaclient/chains/bitcoin/observer/event.go

```diff
@@ -31,6 +34,9 @@ type BTCInboundEvent struct {
 	// Value is the amount of BTC
 	Value float64
 
+	// MsgVoteAmount is the amount used in inbound vote message
+	MsgVoteAmount cosmosmath.Uint
```

Comment:
The name is confusing.
We have a convention `Msg[something]` is for Cosmos message, and this might confusing reader.

We should use a name like
```
AmountForMsgVoteInbound
```
something like that